frontend/src/api/LogApi.js

```diff
@@ -27,3 +27,7 @@ export async function getAllLogs() {
 export async function getDeviceLogs(deviceSerialNumber) {
   return getLogs(`${process.env.REACT_APP_LOG}/${deviceSerialNumber}`);
 }
+
+export async function getStreamLogs(streamId) {
+  return getLogs(`${process.env.REACT_APP_STREAMLOGS}/${streamId}`);
+}
```

Comment:
Stream logs are not in the same format as other logs. The existing `LogInfo` object will not be sufficient. You'll have to make a new `StreamLogInfo` with additional information, and display all the info in the table.

- Log ID
- Stream ID
- Encoder Serial Number
- Decoder Serial Number
- Log level
- Message
- Date

Here is the format of what you get from the backend, you'll have to `.map` it to get the data you need to make your streamloginfo.
```
[
    {
        "id": 26,
        "serialNumber": "d7TxFn7o", //This is serial number of the sender
        "streamId": 1,
        "logEntity": {
            "id": 26,
            "dateTime": "2021-04-04T17:21:38.636972Z",
            "message": "A stream started from output channel 1 of decoder panda to input channel 7 of encoder d7TxFn7o",
            "level": "info",
            "serialNumber": "panda" //This is the serial number of the receiver 
        }
    }
]
```

Reply:
I'll create a new model in that case. Thanks for elaborating on the expected format!